client/admin/brewUtils/brewLookup/brewLookup.jsx

```diff
@@ -12,22 +14,45 @@ const BrewLookup = createClass({
 	},
 	getInitialState() {
 		return {
-			query     : '',
-			foundBrew : null,
-			searching : false,
-			error     : null
+			query          : '',
+			foundBrew      : null,
+			searching      : false,
+			error          : null,
+			checkForScript : false,
+			scriptCount    : undefined
 		};
 	},
 	handleChange(e){
 		this.setState({ query: e.target.value });
 	},
 	lookup(){
-		this.setState({ searching: true, error: null });
+		this.setState({ searching: true, error: null, checkForScript: false, scriptCount: undefined });
 
 		request.get(`/admin/lookup/${this.state.query}`)
+			.then((res)=>{
+				const foundBrew = res.body;
+				const scriptCheck = foundBrew?.text.match(/(<\/?s)cript/g);
+				this.setState({
+					foundBrew      : foundBrew,
+					scriptCount    : scriptCheck?.length || 0,
+					checkForScript : scriptCheck?.length > 0
+				});
+			})
+			.catch((err)=>this.setState({ error: err }))
+			.finally(()=>{
+				this.setState({
+					searching : false
+				});
+			});
+	},
+
+	cleanScript(){
```

Comment:
Let's pull this out so we aren't defining a function inside of another function.

Reply:
Which function are we pulling out here?

Reply:
I've pulled the counting function out to a separate function call now, and it gets properly called after the brew text is updated. This required a minor update to the `updateBrew` function, specifically to the section that removes the `text` field after compressing to `textBin`.

Reply:
> Which function are we pulling out here?

This was referring to `cleanScript` which was defined inside of `lookup`